MinimOsd_Extra/protocols/NMEA.h

```diff
@@ -150,23 +151,22 @@ bool parse_NMEA_char(char c, char *string) {
         switch(param) { // words in sentence
         case 0:    //frame identification
             frame = 0;
-            if (string[0] == 'G' && string[2] == 'G' && string[3] == 'G' && string[4] == 'A') frame = FRAME_GGA;
-            if (string[0] == 'G' && string[2] == 'R' && string[3] == 'M' && string[4] == 'C') frame = FRAME_RMC;
+            if (string[0] == 'G' && string[1] == 'P' && string[2] == 'G' && string[3] == 'G' && string[4] == 'A') frame = FRAME_GGA;
+            if (string[0] == 'G' && string[1] == 'P' && string[2] == 'R' && string[3] == 'M' && string[4] == 'C') frame = FRAME_RMC;
```

Comment:
Do you ever listen about NMEAv4? GNGGA, GLGGA and so on will be filtered out. This if YOUR bug and must be fixed